### DISCUSSION TITLE
Two working controllers

### DISCUSSION THREAD

**Discussion by Steve-J-W** [DCC-EX/EX-T3-WiFi#58] [General]
Hi Alex,
I built your tethered version last year and this WiFi version a couple of weeks ago. Had no problems, just a bit of fettling to get the screen to fit, frame fitted the first time. Was able to wire the components using the images provided. Uploaded the software (original build) and gave it an initial test and all worked fine. Didn’t print the box (6.5 hours) till a couple of days ago and that worked out ok. 

Helped a friend with his today. It is the first time he has used Visio Studio Code and PlatformIO only issue was a wiring problem, worked out ok considering it was via a Zoom call.

At the same time downloaded your latest software and as luck would have it updated Visio Studio Code before reloading the software.

Thanks for all your work on this Steve

**Comment by alex-code** [comment]
Glad you like it and were able to follow my instructions.
I'm working towards a binary build that can be uploaded with a Chromium based web browser so you won't need VSCode and PlatformIO.
